docs/using/api.md

```diff
@@ -255,3 +255,12 @@ Within the jobs list, a complete job dictionary will look like:
 ```
 
 Importantly, the `files` block provides download links for the product files.
+
+For large queries not all jobs may be provided single request. In this case there will be a `next` key in the response that will continue the query (this response may be similarly truncated and include a `next` key).
```

Comment:
`For large queries, all jobs may not be provided...`

Reply:
and specifically call out `next` as a url?  Example might be enough, though.  `there will be a `next` url in the response...`?